### DISCUSSION TITLE
Idea: elevate without installing.

### DISCUSSION THREAD

**Discussion by My1** [rustdesk/rustdesk#1870] [General]
While partially similar to #70 the option of just elevating a portable version doesnt have to rely on remote-entering a password.

remote entering is great when the person in front of the computer is not admin but it might be nice to have a quick-elevate on connect (or even just elevate while connected later on) similar to anydesk where you can allow the application to run on admin level and pass UAC and all through by nature of the local user going through UAC.

and also not everyone might want to install a remote control application permanently anyway.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/2041
